src/ppx/grfx/grfx_pipeline.cpp

```diff
@@ -97,6 +107,21 @@ grfx::BlendAttachmentState BlendAttachmentState::BlendModePremultAlpha()
     return state;
 }
 
+grfx::BlendAttachmentState BlendAttachmentState::BlendModeDisableOutput()
+{
+    grfx::BlendAttachmentState state = {};
+    state.blendEnable                = false;
+    state.srcColorBlendFactor        = grfx::BLEND_FACTOR_ONE;
```

Comment:
sorry my bad in my branch, but we dont need to set all the blendFactor/ops here since blend is disabled anyways